networking/hub-default.json

```diff
@@ -70,6 +70,8 @@
             "[concat(variables('baseFwPipName'), '-02')]"
         ],
         "hubFwName": "[concat('fw-', parameters('location'))]",
+        "fwPoliciesBaseName": "[concat('fw-policies-base')]",
```

Comment:
was something supposed to be `concat`'d here?

Reply:
my bad, I deleted the 'location' , I didn't delete the concat. Then in multicluster, this will be a general base policy on the 'share' resource group.
Good catch. I will fix it.

Reply:
done